pkg/envoy/xds_server.go

```diff
@@ -91,6 +92,8 @@ type Listener struct {
 type XDSServer interface {
 	// AddListener adds a listener to a running Envoy proxy.
 	AddListener(name string, kind policy.L7ParserType, port uint16, isIngress bool, mayUseOriginalSourceAddr bool, wg *completion.WaitGroup)
+	// AddAdminListener adds an Admin API listener to Envoy.
+	AddAdminListener(port uint16, wg *completion.WaitGroup)
```

Comment:
not completely happy with this new API, but we already have another internal and generic addListener